sdk/purview/purview-datamap-rest/recordings/node/my_test/recording_sample_test.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Entries": [],
```

Comment:
Could you migrate your recordings? https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/Quickstart-on-how-to-write-tests.md#how-to-push-test-recordings-to-assets-repo

Reply:
updated